docs/kb/semgrep-ci/github-repository-rulesets-semgrep.md

```diff
@@ -0,0 +1,75 @@
+---
+tags:
+  - GitHub
+  - Repository rulesets
+description: Setting up GitHub required workflows for Semgrep scans.
+---
+
+# Using GitHub repository rulesets to implement Semgrep
+
+If your GitHub organization has hundreds or thousands of repositories, setting up Semgrep by adding the `.github/workflows/semgrep.yml` file to all repositories can be a time-consuming task. You can instead use [repository rulesets](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/creating-rulesets-for-a-repository#introduction) to add Semgrep as a workflow that is [required to pass before merging](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#require-workflows-to-pass-before-merging). 
```

Comment:
```suggestion
Use [GitHub repository rulesets](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/creating-rulesets-for-a-repository#introduction) to quickly implement Semgrep scans across hundreds or thousands of repositories in your GitHub organization. Repository rulesets enable you to add a Semgrep scan to many repositories as a workflow that is [required to pass before merging](https://docs.github.com/en/enterprise-cloud@latest/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#require-workflows-to-pass-before-merging).
```

I feel like there's two things going on here in this paragraph: 
1. Repository rulesets make it easy to add Semgrep scans
2. Repository rulesets require that the workflow must pass. 

We may want to separate (2) as its own paragraph. 

Also, I think there are two things we should mention early -  mainly that "repository rulesets make it easy to add Semgrep scans to PRs by having 1 repository rulesets JSON file exported to many repos" (my understanding is based on the helpful documentation link you provided).